src/main/java/snowy/storage/Storage.java

```diff
@@ -0,0 +1,99 @@
+package snowy.storage;
+
+import snowy.tasklist.Task;
+import snowy.tasklist.TaskList;
+import snowy.data.SnowyException;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Handles saving and loading of tasks from a file.
+ */
```

Comment:
header comment seems too vague for the amount of code the class handles. Could include a more detailed header